# Titanic competition with TensorFlow Decision Forests

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition. 

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.8.1


# Load dataset

In [2]:
train_df = pd.read_csv("train.csv")
serving_df = pd.read_csv("test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [3]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [4]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [5]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

2024-02-07 23:11:30.867921: I metal_plugin/src/device/metal_device.cc:1154] Metal device set to: Apple M2
2024-02-07 23:11:30.867936: I metal_plugin/src/device/metal_device.cc:296] systemMemory: 16.00 GB
2024-02-07 23:11:30.867941: I metal_plugin/src/device/metal_device.cc:313] maxCacheSize: 5.33 GB
2024-02-07 23:11:30.868150: I tensorflow/core/common_runtime/pluggable_device/pluggable_device_factory.cc:306] Could not identify NUMA node of platform GPU ID 0, defaulting to 0. Your kernel may not have been built with NUMA support.
2024-02-07 23:11:30.868466: I tensorflow/core/common_runtime/pluggable_device/pluggable_device_factory.cc:272] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 0 MB memory) -> physical PluggableDevice (device: 0, name: METAL, pci bus id: <undefined>)


# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [6]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-02-07 23:11:31.0076 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:11:31.0083 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:11:31.0084 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
2024-02-07 23:11:32.500154: I tensorflow/core/grappler/optimizers/custom_graph_optimizer_registry.cc:117] Plugin optimizer for device_type GPU is enabled.
[INFO 24-02-07 23:11:32.8507 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpgi6n6xe5/model/ with prefix 1cb2d3ccdb3f4cd7
[INFO 24-02-07 23:11:32.8544 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-07 23:11:32.8544 IST kernel.cc:1061] Use fast generic engine


Accuracy: 0.804347813129425 Loss:0.8922085165977478


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-02-07 23:11:33.4448 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:11:33.4448 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:11:33.4448 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.77173912525177 Loss:1.0136744976043701


[INFO 24-02-07 23:11:33.8365 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp6nm6lh1q/model/ with prefix 002ecd96273a47eb
[INFO 24-02-07 23:11:33.8412 IST decision_forest.cc:660] Model loaded with 54 root(s), 2824 node(s), and 10 input feature(s).
[INFO 24-02-07 23:11:33.8412 IST kernel.cc:1061] Use fast generic engine


Let's look at the model and you can also notice the information about variable importance that the model figured out

In [8]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.431313 ################
    2.           "Age"  0.373986 ############
    3.          "Fare"  0.250774 ####
    4.          "Name"  0.225272 ###
    5.   "Ticket_item"  0.182111 
    6.      "Embarked"  0.181387 
    7. "Ticket_number"  0.180897 
    8.        "Pclass"  0.178279 
    9.         "Parch"  0.175344 
   10.         "SibSp"  0.172167 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 39.00000

# Make predictions

In [9]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)

Submission exported to submission.csv


# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).


In [10]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpmsjyezst as temporary training directory


[WARNING 24-02-07 23:11:34.0467 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:11:34.0467 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:11:34.0467 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


Accuracy: 0.8904109597206116 Loss:0.6763895750045776


[INFO 24-02-07 23:12:01.3885 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpmsjyezst/model/ with prefix f3dc806e1af74d80
[INFO 24-02-07 23:12:01.3967 IST decision_forest.cc:660] Model loaded with 26 root(s), 3162 node(s), and 12 input feature(s).
[INFO 24-02-07 23:12:01.3968 IST abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 24-02-07 23:12:01.3968 IST kernel.cc:1061] Use fast generic engine


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [11]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

[WARNING 24-02-07 23:12:01.5259 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:01.5260 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:01.5260 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:0


[INFO 24-02-07 23:12:01.7990 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp5x11b4c_/model/ with prefix ce32390de2564580
[INFO 24-02-07 23:12:01.8006 IST kernel.cc:1061] Use fast generic engine


[WARNING 24-02-07 23:12:01.9764 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:01.9764 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:01.9764 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:1


[INFO 24-02-07 23:12:02.2324 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpa_kw7b2f/model/ with prefix d0abbf8f6f99408f
[INFO 24-02-07 23:12:02.2360 IST kernel.cc:1061] Use fast generic engine


[WARNING 24-02-07 23:12:02.4089 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:02.4089 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:02.4089 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:2


[INFO 24-02-07 23:12:02.6654 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpbytm0gg1/model/ with prefix ef683c751d4e4eee
[INFO 24-02-07 23:12:02.6692 IST kernel.cc:1061] Use fast generic engine


[WARNING 24-02-07 23:12:02.8447 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:02.8447 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:02.8447 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:3


[INFO 24-02-07 23:12:03.1008 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpxrs3i1q1/model/ with prefix 44fe06279e524c76
[INFO 24-02-07 23:12:03.1036 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:03.2737 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:03.2738 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:03.2738 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:4


[INFO 24-02-07 23:12:03.4652 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpx4506ka6/model/ with prefix 4b1b5a720da44757
[INFO 24-02-07 23:12:03.4668 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:03.6377 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:03.6377 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:03.6377 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:5


[INFO 24-02-07 23:12:03.8458 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpfg6nipd7/model/ with prefix 27b0bd70f24340c5
[INFO 24-02-07 23:12:03.8477 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:04.0179 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:04.0179 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:04.0179 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:6


[INFO 24-02-07 23:12:04.2584 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmphr6c3dta/model/ with prefix 6a82dbae638c49e6
[INFO 24-02-07 23:12:04.2616 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:04.4296 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:04.4296 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:04.4296 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:7


[INFO 24-02-07 23:12:04.6422 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpvh1upszo/model/ with prefix 2212b08808ce4f82
[INFO 24-02-07 23:12:04.6444 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:04.8136 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:04.8137 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:04.8137 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:8


[INFO 24-02-07 23:12:05.0793 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpgtc46hh5/model/ with prefix 6e2d86ccf9964803
[INFO 24-02-07 23:12:05.0834 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:05.2521 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:05.2521 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:05.2521 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:9


[INFO 24-02-07 23:12:05.5357 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp86mwu0u1/model/ with prefix c8c2793dcd3a4f37
[INFO 24-02-07 23:12:05.5404 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:05.7116 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:05.7116 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:05.7116 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:10


[INFO 24-02-07 23:12:05.9335 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpv0r9dib6/model/ with prefix 2d861dde446540f0
[INFO 24-02-07 23:12:05.9359 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:06.1066 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:06.1067 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:06.1067 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:11


[INFO 24-02-07 23:12:06.4008 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmptc683p68/model/ with prefix ebc2bda298724d80
[INFO 24-02-07 23:12:06.4059 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:06.5844 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:06.5844 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:06.5844 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:12


[INFO 24-02-07 23:12:06.8409 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpt4d6u4w2/model/ with prefix b258c787c51e4c6e
[INFO 24-02-07 23:12:06.8439 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:07.0115 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:07.0115 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:07.0115 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:13


[INFO 24-02-07 23:12:07.4342 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkb7tfu6v/model/ with prefix 65e0d58819714a11
[INFO 24-02-07 23:12:07.4434 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:07.6124 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:07.6125 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:07.6125 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:14


[INFO 24-02-07 23:12:07.8545 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpl8qwtdzm/model/ with prefix 37728b03c7e54b71
[INFO 24-02-07 23:12:07.8575 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:08.0304 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:08.0304 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:08.0304 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:15


[INFO 24-02-07 23:12:08.2327 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpia1zy25e/model/ with prefix 0212ed92720c46f5
[INFO 24-02-07 23:12:08.2344 IST kernel.cc:1061] Use fast generic engine


i:16


[WARNING 24-02-07 23:12:08.5670 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:08.5670 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:08.5671 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:08.8202 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpvag3byfw/model/ with prefix 2d6574737a50493e
[INFO 24-02-07 23:12:08.8239 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:09.0061 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:09.0061 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:09.0061 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:17


[INFO 24-02-07 23:12:09.2559 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0mc7mw45/model/ with prefix 38c4241034d449ca
[INFO 24-02-07 23:12:09.2581 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:09.4362 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:09.4362 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:09.4362 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:18


[INFO 24-02-07 23:12:09.7076 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjlj5brbe/model/ with prefix 0987437116564296
[INFO 24-02-07 23:12:09.7118 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:09.8821 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:09.8821 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:09.8821 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:19


[INFO 24-02-07 23:12:10.1405 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp2plgybar/model/ with prefix 4bd5c3d1da474818
[INFO 24-02-07 23:12:10.1441 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:10.3187 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:10.3188 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:10.3188 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:20


[INFO 24-02-07 23:12:10.5359 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpr76o_9v5/model/ with prefix b4ad1e772b4545f4
[INFO 24-02-07 23:12:10.5380 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:10.7076 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:10.7076 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:10.7076 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:21


[INFO 24-02-07 23:12:10.9279 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp4gx_hh60/model/ with prefix 9b73eb7df9f347ac
[INFO 24-02-07 23:12:10.9302 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:11.0979 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:11.0979 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:11.0979 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:22


[INFO 24-02-07 23:12:11.3395 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpef0ihhr0/model/ with prefix 273fcd14a7c84b7d
[INFO 24-02-07 23:12:11.3425 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:11.5186 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:11.5186 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:11.5186 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:23


[INFO 24-02-07 23:12:11.7505 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpq2dn3f2h/model/ with prefix 20638e6d07514b9f
[INFO 24-02-07 23:12:11.7528 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-07 23:12:11.7528 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:11.9235 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:11.9236 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:11.9236 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:24


[INFO 24-02-07 23:12:12.1350 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpxs5e882_/model/ with prefix e8a1fbda9799438d
[INFO 24-02-07 23:12:12.1372 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:12.3084 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:12.3084 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:12.3084 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:25


[INFO 24-02-07 23:12:12.5512 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpizdyjh6u/model/ with prefix e3fa7240e5e84ab8
[INFO 24-02-07 23:12:12.5541 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:12.7242 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:12.7243 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:12.7243 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:26


[INFO 24-02-07 23:12:12.9301 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpe06vl_zt/model/ with prefix d17fddd3a54845f4
[INFO 24-02-07 23:12:12.9319 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:13.1044 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:13.1045 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:13.1045 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:27


[INFO 24-02-07 23:12:13.3664 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpj88my40l/model/ with prefix 1d5267d1540442ef
[INFO 24-02-07 23:12:13.3705 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:13.5388 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:13.5388 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:13.5388 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:28


[INFO 24-02-07 23:12:13.7263 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpeh1aym82/model/ with prefix 464f525a3691463a
[INFO 24-02-07 23:12:13.7275 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:13.8961 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:13.8961 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:13.8961 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:29


[INFO 24-02-07 23:12:14.1039 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp230bs0jm/model/ with prefix 7f5a2d21f3c94b8b
[INFO 24-02-07 23:12:14.1059 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:14.2765 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:14.2765 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:14.2765 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:30


[INFO 24-02-07 23:12:14.5350 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpepc54bg_/model/ with prefix 9072e6d1010e4947
[INFO 24-02-07 23:12:14.5386 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:14.7082 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:14.7082 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:14.7082 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:31


[INFO 24-02-07 23:12:15.0279 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpds95cxt_/model/ with prefix b3fa339025b5413c
[INFO 24-02-07 23:12:15.0332 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:15.2042 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:15.2043 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:15.2043 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:32


[INFO 24-02-07 23:12:15.4280 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpzgjr13zq/model/ with prefix a24dc787b7a9418c
[INFO 24-02-07 23:12:15.4305 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:15.5995 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:15.5995 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:15.5995 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:33


[INFO 24-02-07 23:12:15.8402 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpt4qza90y/model/ with prefix 49f26ec09f1f44ec
[INFO 24-02-07 23:12:15.8431 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:16.0125 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:16.0125 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:16.0125 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:34


[INFO 24-02-07 23:12:16.2241 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjwy5arf6/model/ with prefix deaf94f1c1704718
[INFO 24-02-07 23:12:16.2263 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:16.3984 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:16.3985 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:16.3985 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:35


[INFO 24-02-07 23:12:16.7240 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp330iqjws/model/ with prefix 0259756f2b7b489d
[INFO 24-02-07 23:12:16.7274 IST kernel.cc:1061] Use fast generic engine


i:36


[WARNING 24-02-07 23:12:16.9254 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:16.9254 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:16.9255 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:17.1417 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpbw4rtx52/model/ with prefix dab71b7a29264386
[INFO 24-02-07 23:12:17.1439 IST kernel.cc:1061] Use fast generic engine


i:37


[WARNING 24-02-07 23:12:17.5539 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:17.5540 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:17.5540 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:17.7752 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpatmovvxj/model/ with prefix ee99da5318ca4834
[INFO 24-02-07 23:12:17.7773 IST kernel.cc:1061] Use fast generic engine


i:38


[WARNING 24-02-07 23:12:17.9784 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:17.9784 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:17.9784 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:18.2278 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmphlqf1lkf/model/ with prefix e42fb4199e1d4a18
[INFO 24-02-07 23:12:18.2311 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:18.4127 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:18.4127 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:18.4127 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:39


[INFO 24-02-07 23:12:18.6882 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjc0vaeua/model/ with prefix b4c3bad9b95c4d5b
[INFO 24-02-07 23:12:18.6923 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:18.8828 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:18.8829 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:18.8829 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:40


[INFO 24-02-07 23:12:19.1224 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp7c43eb31/model/ with prefix a78290ec5fea4516
[INFO 24-02-07 23:12:19.1248 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:19.2971 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:19.2971 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:19.2971 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:41


[INFO 24-02-07 23:12:19.5671 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpxt296k2q/model/ with prefix 16b2541f7d424a41
[INFO 24-02-07 23:12:19.5707 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:19.7487 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:19.7488 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:19.7488 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:42


[INFO 24-02-07 23:12:19.9842 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpg2c4k1g9/model/ with prefix 34b49e1db00e4b25
[INFO 24-02-07 23:12:19.9867 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:20.1572 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:20.1572 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:20.1572 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:43


[INFO 24-02-07 23:12:20.4947 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp1oggk76d/model/ with prefix 2d9b9942920546fc
[INFO 24-02-07 23:12:20.4980 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:20.6719 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:20.6720 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:20.6720 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:44


[INFO 24-02-07 23:12:20.8970 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjlxbdh8v/model/ with prefix 43da1b6db6f3445a
[INFO 24-02-07 23:12:20.8995 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:21.0889 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:21.0889 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:21.0889 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:45


[INFO 24-02-07 23:12:21.2831 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpgon7g9yd/model/ with prefix 8009642e577d4c02
[INFO 24-02-07 23:12:21.2843 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:21.4662 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:21.4663 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:21.4663 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:46


[INFO 24-02-07 23:12:21.7486 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp32q91jnf/model/ with prefix 0632e13b516a4d34
[INFO 24-02-07 23:12:21.7518 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:21.9449 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:21.9449 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:21.9449 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:47


[INFO 24-02-07 23:12:22.1810 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmptf0ao7p0/model/ with prefix f0fb002e5690473d
[INFO 24-02-07 23:12:22.1833 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-07 23:12:22.1833 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:22.3697 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:22.3697 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:22.3697 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:48


[INFO 24-02-07 23:12:22.5924 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpcvgs1kh5/model/ with prefix 64f309154a214a80
[INFO 24-02-07 23:12:22.5960 IST kernel.cc:1061] Use fast generic engine


i:49


[WARNING 24-02-07 23:12:22.7978 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:22.7979 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:22.7979 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:23.0984 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpbv8wdr9e/model/ with prefix 3718ab24bb40403c
[INFO 24-02-07 23:12:23.1012 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:23.2786 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:23.2786 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:23.2786 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:50


[INFO 24-02-07 23:12:23.5615 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkf1wqpf0/model/ with prefix 62eeee8da0394fcc
[INFO 24-02-07 23:12:23.5659 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:23.7612 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:23.7613 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:23.7613 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:51


[INFO 24-02-07 23:12:24.0615 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp1o4w2sht/model/ with prefix 9516eb1aff4e4cef
[INFO 24-02-07 23:12:24.0663 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:24.2403 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:24.2403 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:24.2403 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:52


[INFO 24-02-07 23:12:24.4974 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp61r4hdrd/model/ with prefix f8482fa6deee4d6d
[INFO 24-02-07 23:12:24.5000 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:24.6871 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:24.6871 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:24.6871 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:53


[INFO 24-02-07 23:12:24.9233 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkh69z5ll/model/ with prefix 79ca1458a75a4331
[INFO 24-02-07 23:12:24.9260 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:25.1085 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:25.1085 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:25.1085 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:54


[INFO 24-02-07 23:12:25.3460 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpj67c0jsj/model/ with prefix 8c8ee30b78c84d1e
[INFO 24-02-07 23:12:25.3486 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:25.5328 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:25.5328 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:25.5329 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:55


[INFO 24-02-07 23:12:25.7832 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp5rx6jatw/model/ with prefix 776f498948d0414b
[INFO 24-02-07 23:12:25.7864 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:25.9602 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:25.9602 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:25.9602 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:56


[INFO 24-02-07 23:12:26.2198 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpt3zrx5aj/model/ with prefix bab35718ef53460a
[INFO 24-02-07 23:12:26.2224 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:26.3973 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:26.3973 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:26.3973 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:57


[INFO 24-02-07 23:12:26.6053 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpal59qxgo/model/ with prefix 8d39ea1bbf924652
[INFO 24-02-07 23:12:26.6069 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:26.7943 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:26.7943 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:26.7943 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:58


[INFO 24-02-07 23:12:27.1487 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmprmjen_8d/model/ with prefix ffe747e30ad84c85
[INFO 24-02-07 23:12:27.1541 IST kernel.cc:1061] Use fast generic engine


i:59


[WARNING 24-02-07 23:12:27.3555 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:27.3556 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:27.3556 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:27.6872 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp24f9uuv_/model/ with prefix c1f9723f938d43a0
[INFO 24-02-07 23:12:27.6920 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:27.8712 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:27.8712 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:27.8712 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:60


[INFO 24-02-07 23:12:28.0916 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpf_lq5fiz/model/ with prefix a2919d5acc854944
[INFO 24-02-07 23:12:28.0933 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:28.2782 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:28.2783 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:28.2783 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:61


[INFO 24-02-07 23:12:28.7444 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp13pi3_07/model/ with prefix c9cb49b39b7c429b
[INFO 24-02-07 23:12:28.7471 IST kernel.cc:1061] Use fast generic engine


i:62


[WARNING 24-02-07 23:12:28.9662 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:28.9662 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:28.9662 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:29.2141 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpw6f0vd1s/model/ with prefix 5cf992ce1c8a4853
[INFO 24-02-07 23:12:29.2171 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:29.3932 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:29.3932 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:29.3933 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:63


[INFO 24-02-07 23:12:29.6163 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpultnd4t5/model/ with prefix 8190b8dae1404a0b
[INFO 24-02-07 23:12:29.6203 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:29.8093 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:29.8093 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:29.8093 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:64


[INFO 24-02-07 23:12:30.0460 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkwafg5fl/model/ with prefix e294cd0eb6c2413a
[INFO 24-02-07 23:12:30.0487 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:30.2442 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:30.2442 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:30.2442 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:65


[INFO 24-02-07 23:12:30.5156 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp508pxg56/model/ with prefix eb6dfcd0db534634
[INFO 24-02-07 23:12:30.5191 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:30.6950 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:30.6950 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:30.6950 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:66


[INFO 24-02-07 23:12:30.9320 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmps162fr74/model/ with prefix 974227b444324e42
[INFO 24-02-07 23:12:30.9339 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:31.1194 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:31.1195 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:31.1195 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:67


[INFO 24-02-07 23:12:31.3597 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp89gihh_p/model/ with prefix 46bda732b21a475d
[INFO 24-02-07 23:12:31.3625 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:31.5466 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:31.5466 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:31.5466 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:68


[INFO 24-02-07 23:12:31.9023 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpnot60tat/model/ with prefix 42f87346bf0d46ab
[INFO 24-02-07 23:12:31.9061 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:32.0801 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:32.0801 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:32.0801 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:69


[INFO 24-02-07 23:12:32.3496 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmppqsldwoh/model/ with prefix 0fd3aadc9ee14f1a
[INFO 24-02-07 23:12:32.3532 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-07 23:12:32.3533 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:32.5279 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:32.5280 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:32.5280 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:70


[INFO 24-02-07 23:12:32.7469 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp1mulmvp4/model/ with prefix a1715773f9bf46ff
[INFO 24-02-07 23:12:32.7492 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:32.9254 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:32.9254 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:32.9255 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:71


[INFO 24-02-07 23:12:33.1405 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpuyweedto/model/ with prefix 181c7bba50f54c40
[INFO 24-02-07 23:12:33.1422 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:33.3231 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:33.3232 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:33.3232 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:72


[INFO 24-02-07 23:12:33.5750 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjpddry1k/model/ with prefix 02597778fa36442a
[INFO 24-02-07 23:12:33.5777 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:33.7515 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:33.7515 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:33.7515 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:73


[INFO 24-02-07 23:12:33.9576 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp93i3rfji/model/ with prefix 9e76421a02ee4677
[INFO 24-02-07 23:12:33.9590 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:34.1536 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:34.1536 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:34.1536 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:74


[INFO 24-02-07 23:12:34.4736 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjuf0fhz2/model/ with prefix 70f3c147544342d4
[INFO 24-02-07 23:12:34.4787 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:34.6595 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:34.6595 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:34.6595 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:75


[INFO 24-02-07 23:12:34.8771 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmps_n3z_dj/model/ with prefix 819ff005c6e54219
[INFO 24-02-07 23:12:34.8789 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:35.0555 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:35.0555 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:35.0555 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:76


[INFO 24-02-07 23:12:35.2850 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp3ov_rqy8/model/ with prefix 917389cb1a87450b
[INFO 24-02-07 23:12:35.2868 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:35.4691 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:35.4691 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:35.4691 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:77


[INFO 24-02-07 23:12:35.7124 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp66tf7om9/model/ with prefix dd8a879ef1514c04
[INFO 24-02-07 23:12:35.7154 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:35.9033 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:35.9034 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:35.9034 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:78


[INFO 24-02-07 23:12:36.1072 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmplto7w8wx/model/ with prefix 004fa670229e4d08
[INFO 24-02-07 23:12:36.1085 IST kernel.cc:1061] Use fast generic engine


i:79


[WARNING 24-02-07 23:12:36.3076 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:36.3076 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:36.3076 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:36.5873 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmph0f54g9q/model/ with prefix d0bd15ce74e6497e
[INFO 24-02-07 23:12:36.5910 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:36.7824 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:36.7824 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:36.7824 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:80


[INFO 24-02-07 23:12:37.0647 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpcazdu325/model/ with prefix e439963031604507
[INFO 24-02-07 23:12:37.0668 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:37.2461 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:37.2461 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:37.2461 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:81


[INFO 24-02-07 23:12:37.5131 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmphchlmopn/model/ with prefix f2660981723f4378
[INFO 24-02-07 23:12:37.5162 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:37.7015 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:37.7015 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:37.7015 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:82


[INFO 24-02-07 23:12:37.9694 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0madwj53/model/ with prefix f08886f20f3e4fa4
[INFO 24-02-07 23:12:37.9730 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:38.1579 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:38.1579 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:38.1579 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:83


[INFO 24-02-07 23:12:38.4450 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpjcd38kzs/model/ with prefix 6b7b172d91e14fa8
[INFO 24-02-07 23:12:38.4486 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:38.6251 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:38.6251 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:38.6251 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:84


[INFO 24-02-07 23:12:38.8968 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpn0v0vhps/model/ with prefix 6ca2a601d7214858
[INFO 24-02-07 23:12:38.9000 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:39.0832 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:39.0832 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:39.0832 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:85


[INFO 24-02-07 23:12:39.3647 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpcz2rd1zo/model/ with prefix 545747b0a7534f7e
[INFO 24-02-07 23:12:39.3675 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:39.5648 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:39.5648 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:39.5648 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:86


[INFO 24-02-07 23:12:39.9145 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp5nifohh0/model/ with prefix 3ba5415552b64c91
[INFO 24-02-07 23:12:39.9195 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:40.0959 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:40.0959 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:40.0959 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:87


[INFO 24-02-07 23:12:40.3765 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp85xh2wge/model/ with prefix bc7bfab915514e4a
[INFO 24-02-07 23:12:40.3790 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:40.5527 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:40.5527 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:40.5527 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:88


[INFO 24-02-07 23:12:40.7599 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpd1s9gs9x/model/ with prefix f561e5300d944c2b
[INFO 24-02-07 23:12:40.7614 IST kernel.cc:1061] Use fast generic engine


i:89


[WARNING 24-02-07 23:12:41.2200 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:41.2201 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:41.2201 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:41.5011 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmps0z925r6/model/ with prefix 895e1dab938e419b
[INFO 24-02-07 23:12:41.5041 IST kernel.cc:1061] Use fast generic engine


i:90


[WARNING 24-02-07 23:12:41.7098 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:41.7098 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:41.7098 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-02-07 23:12:41.9694 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpkz8wv0jz/model/ with prefix 13b92e04e8a34fd0
[INFO 24-02-07 23:12:41.9721 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:42.1506 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:42.1506 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:42.1506 IST gradient_boosted_trees.cc:1911] "selective_gradient_boostin

i:91


[INFO 24-02-07 23:12:42.4236 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp0pcxfepc/model/ with prefix 7087e06a8c3e4bb1
[INFO 24-02-07 23:12:42.4275 IST abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-02-07 23:12:42.4275 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:42.6064 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:42.6064 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:42.6064 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:92


[INFO 24-02-07 23:12:42.9695 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpstrc58i3/model/ with prefix 6348eed68b6b439b
[INFO 24-02-07 23:12:42.9763 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:43.1599 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:43.1599 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:43.1600 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:93


[INFO 24-02-07 23:12:43.3747 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpcbydk2lt/model/ with prefix 1d340425fc694b92
[INFO 24-02-07 23:12:43.3766 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:43.5567 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:43.5568 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:43.5568 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:94


[INFO 24-02-07 23:12:43.7964 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpryvujk_f/model/ with prefix 175f189b28574430
[INFO 24-02-07 23:12:43.7988 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:43.9834 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:43.9834 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:43.9834 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:95


[INFO 24-02-07 23:12:44.2313 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpl6lchl94/model/ with prefix fce4a62661d84763
[INFO 24-02-07 23:12:44.2338 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:44.4132 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:44.4132 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:44.4132 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:96


[INFO 24-02-07 23:12:44.6732 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmp_8o_52v3/model/ with prefix 49011da3a2fb4129
[INFO 24-02-07 23:12:44.6761 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:44.8515 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:44.8515 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:44.8515 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:97


[INFO 24-02-07 23:12:45.0710 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpleom0q4r/model/ with prefix 4d870cdddadd4e8f
[INFO 24-02-07 23:12:45.0731 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:45.2592 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:45.2592 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:45.2592 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:98


[INFO 24-02-07 23:12:45.6558 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpszukq_ne/model/ with prefix 44eb5d9fed524c98
[INFO 24-02-07 23:12:45.6629 IST kernel.cc:1061] Use fast generic engine
[WARNING 24-02-07 23:12:45.8416 IST gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:45.8416 IST gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-02-07 23:12:45.8416 IST gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


i:99
Submission exported to submission.csv


[INFO 24-02-07 23:12:46.3025 IST kernel.cc:1233] Loading model from path /var/folders/17/bs5_4rn1063fk_n8_114kqdw0000gn/T/tmpv_iq5my8/model/ with prefix 5ca945707e6949e5
[INFO 24-02-07 23:12:46.3123 IST kernel.cc:1061] Use fast generic engine


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests) 